pkg/event/dispatcher.go

```diff
@@ -65,6 +65,17 @@ func (ed *HTTPEventDispatcher) DispatchEvent(event LogEvent) (bool, error) {
 	return success, err
 }
 
+func NewHttpEventDispatcher(sdkKey string, requester *utils.HTTPRequester, logger logging.OptimizelyLogProducer) Dispatcher {
```

Comment:
Need to be consistent with naming and make `HTTP` all caps here since you have it for the requester factory method below